docs/examples.md

```diff
@@ -36,13 +36,20 @@ def read_json(path: Path) -> Dict[str, Any]:
         return json.load(f)
 
 
-def plot(name: str) -> None:
+def plot(
+    name: str, autofix_winding: bool = True, suptitle: Optional[str] = None
```

Comment:
maybe name this "fix_winding" instead of "autofix_winding", since the "auto" is  implied?